### ISSUE TITLE
stylelint-config-standard-less rules don't recognize valid Less syntax

### ISSUE THREAD

**Issue by Jagget** [stylelint-less/stylelint-less#104]
# stylelint-config-standard-less rules don't recognize valid Less syntax

Several rules from `stylelint-config-standard-less` produce false positives on valid Less syntax.

## `at-rule-prelude-no-invalid`

Flags Less reference imports as invalid:

```less
@import (reference) './variables.less';
```

Error: `Unexpected invalid prelude "(reference) './variables.less'" for at-rule "@import"`

## `declaration-property-value-no-unknown`

Doesn't recognize Less variables in property values:

```less
padding: 0 @spacer-4;
border: 1px solid @blue;
transform: translate(-@spacer-3, 0);
width: calc(100% + @spacer-3 + @spacer-3);
```

Error: `Cannot parse property value "0 @spacer-4" for property "padding"`

## `container-name-pattern`

Doesn't recognize Less variables in container queries:

```less
@container @container-name (width < @container-sm) { ... }
```

Error: `Expected container name "@container-name" to be kebab-case`

## Environment

- stylelint-config-standard-less: 4.0.1
- stylelint: 16.26.1

## Workaround

Currently disabling these rules:

```js
'at-rule-prelude-no-invalid': null,
'declaration-property-value-no-unknown': null,
'container-name-pattern': null,
```
